src/stop.sh

```diff
@@ -28,6 +28,7 @@ else
   CEPH_CONF_PATH="$PWD"
 fi
 conf_fn="$CEPH_CONF_PATH/ceph.conf"
+CEPHADM_DIR_PATH="$CEPH_CONF_PATH/../src/cephadm"
```

Comment:
This is going to make it even more complicated to have anything but one big cephadm script without any form of packaging.